src/ResourceManager/Network/Commands.Network/help/Update-AzureRmP2SVpnGateway.md

```diff
@@ -0,0 +1,249 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version:
+schema: 2.0.0
+---
+
+# Update-AzureRmP2SVpnGateway
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+### ByP2SVpnGatewayName (Default)
+```
+Update-AzureRmP2SVpnGateway -ResourceGroupName <String> -Name <String> [-VpnGatewayScaleUnit <UInt32>]
```

Comment:
please regen, these are not the right parameter sets.

Reply:
Done.

Reply:
Regenerated help files. Let me know if u still see issues.